lua/neogit/popups/init.lua

```diff
@@ -25,16 +25,12 @@ end
 --- Returns an array useful for creating mappings for the available popups
 ---@return table<string, Mapping>
 function M.mappings_table()
-  local config = require("neogit.config")
-
   return {
     {
       "HelpPopup",
       "Help",
       M.open("help", function(f)
```

Comment:
Do we still need to pass the second argument here, since it's just an empty table now?

Reply:
Like just having `open("help")` without the function as the second arg

Reply:
No, it appears we don't need it. I'll push a commit removing the `env` arg from help as it is now unnecessary with the removal of the magit keybinding.